demos/smart_framing_demo/cpp_gapi/src/custom_kernels.cpp

```diff
@@ -0,0 +1,287 @@
+
+                    //double height = std::exp(output_blob[box_index + 3 * entriesNum]) * region.anchors[2 * n + 1] * original_im_h / scaleH;
+                    //double width = std::exp(output_blob[box_index + 2 * entriesNum]) * region.anchors[2 * n] * original_im_w / scaleW;
+                    double height = std::exp(output_blob[box_index + 3 * entriesNum]) * blobs_anchor[blobID][2 * n + 1] * original_im_h / scaleH;
+                    double width = std::exp(output_blob[box_index + 2 * entriesNum]) * blobs_anchor[blobID][2 * n] * original_im_w / scaleW;
+
+                    custom::DetectedObject obj;
+                    obj.x = static_cast<float>(std::max((x - width / 2), 0.));
+                    obj.y = static_cast<float>(std::max((y - height / 2), 0.));
+                    obj.width = std::min(static_cast<float>(width), original_im_w - obj.x);
+                    obj.height = std::min(static_cast<float>(height), original_im_h - obj.y);
+
+                    for (int j = 0; j < /*region.classes*/ region_classes; ++j) {
+                        int class_index = calculateEntryIndex(entriesNum, /*region.coords*/ region_coords, /*region.classes*/ region_classes, n * entriesNum + i, /*region.coords*/ region_coords + 1 + j);
+                        float prob = scale * postprocessRawData(output_blob[class_index]);
+
+                        //--- Checking confidence threshold conformance and adding region to the list
+                        if (prob >= confidenceThreshold) {
+                            obj.confidence = prob;
+                            obj.labelID = j;
+                            obj.label = getLabelName(obj.labelID);
+                            objects.push_back(obj);
+                        }
+                    }
+                }
+            }
+        }
+    }
+};
+
+using GPostProc = cv::GOpaque<YOLOv4TinyPostProcessing>;
+
+} // anonymous namespace
+
+
+GAPI_OCV_KERNEL(OCVYOLOv4TinyPostProcessing, custom::GYOLOv4TinyPostProcessingKernel) {
+    static void run(const cv::Mat & image, const cv::Mat & in_blob26x26, const cv::Mat & in_blob13x13,
+        const float confidenceThreshold, const float boxIOUThreshold, const bool useAdvancedPostprocessing, std::vector<custom::DetectedObject> &objects) {
+        YOLOv4TinyPostProcessing post_processor;
+        int blob_size[2];
+        slog::debug << "in_blob26x26 size " << in_blob26x26.size << slog::endl;
+        slog::debug << "in_blob13x13 size " << in_blob13x13.size << slog::endl;
+        blob_size[0] = in_blob26x26.size[0] * in_blob26x26.size[1] * in_blob26x26.size[2] * in_blob26x26.size[3];
+        blob_size[1] = in_blob13x13.size[0] * in_blob13x13.size[1] * in_blob13x13.size[2] * in_blob13x13.size[3];
+        int total_size = blob_size[0] + blob_size[1];
+        slog::debug << "total_size " << total_size << slog::endl; //should be 215475
+
+        //OMZ Post-processing for Yolo
```

Comment:
Does OMZ have this postproc in common variant?

Reply:
I don't think so.